tests/storage/iso/test_local_iso_sr.py

```diff
@@ -42,11 +43,13 @@ def test_create_and_destroy_sr(self, host, formatted_and_mounted_ext4_disk):
 @pytest.mark.usefixtures("local_iso_sr")
 class TestLocalISOSR:
 
-    def test_iso_mount_and_read(self, host, local_iso_sr, running_unix_vm):
+    def test_iso_mount_and_read(self, host, local_iso_sr, imported_vm):
```

Comment:
The `unix_vm` fixture would be more appropriate to replace the `running_unix_vm` fixture. Else we lose an important part of the fixture.